cmd/tf-operator.v1/app/options/options.go

```diff
@@ -16,10 +16,13 @@ package options
 
 import (
 	"flag"
+	"time"
 
 	"k8s.io/api/core/v1"
 )
 
+const DefaultResyncPeriod = 12 * time.Hour
```

Comment:
What should be the ideal default resync period?

Reply:
In k8s,  the default resync period is 12 hour. tf-operator is event driven to reach the desired state, so we do not really need the resync mechanism. If the resync period is short, it may reduce the processing speed of tf-operator, so we can make it longer like k8s.

Reply:
The resync is strongly suggested by jeremy. But I think we do not really need it. 12h LGTM. If there are some problems caused by the long period, it should be somewhere has bugs in the implementation.

Reply:
Agree.  But since we are moving to v1,  we need to test well to see if there are any hidden bugs.